docs/standards/smart-contracts/01-introduction.md

```diff
@@ -0,0 +1,91 @@
+- **[LSP1UniversalReceiverDelegateUP](./lsp1-universal-receiver-delegate-up.md)**: The contract that allows the account to react to the calls that it receives (Normal transaction, Token transfer, Vaults transfer etc .. ).
+- **[LSP6KeyManager](./lsp6-key-manager.md)**: The contract that allows a better **multi-control** over the account along with different permissions.
+
+### Digtal Assets (Tokens) and NFT 2.0
+The **Digtal Assets and NFT 2.0** contracts are the newest advanced version of the token standards with many features that enhance the securiy and the overall user experience, as well as compatibility with [ERC725Accounts](../universal-profile/LSP0-Foundation) and [universal recievers](../generic-standards/lsp1-universal-receiver).
+- **[LSP4DigitalAsset-Metadata](./lsp4-digital-asset-metadata)**: The contract that sets the **DigitalAsset-Metadata**.
+- **[LSP7DigitalAsset](./lsp7-digital-asset.md)**: The contract that can represent either fungible or non-fungible tokens (NFTs).
+- **[LSP8IdentifiableDigitalAsset](./lsp8-identifiable-digital-asset.md)**: The contract that represent an non-fungible token (NFTs), with a **bytes32 tokenId** to allow many uses of token identification including numbers, contract addresses, and hashed values (ie. serial numbers).
+
+### Periphery 
+These contracts are not just related to one specific section but could be used together with the **UniversalProfile** and **Digital Asset and NFT 2.0** contracts.
+- **[LSP9Vault](./lsp9-vault.md)**: The contract that represent a **Vault** able to execute and hold assets, could be owned by a **LSP0ERC725Account** contract.
+- **[LSP1UniversalReceiverDelegateVault](./lsp1-universal-receiver-delegate-vault.md)**: The contract that allow the vault to react to the calls it receives (Normal transaction, Token transfer, etc .. ).
+
+
+## Installation
+
+```bash
+npm install @lukso/universalprofile-smart-contracts
+```
+
+## Usage
+
+
+### Create your own UniversalProfile
+
+```solidity
+// MyUP.sol
+// SPDX-License-Identifier: MIT
+
+import "@lukso/universalprofile-smart-contracts/contracts/UniversalProfile.sol";
+
+pragma solidity ^0.8.0;
+
+contract MyUP is UniversalProfile {
+
+    constructor() UniversalProfile(msg.sender) {
+        // ..
+    }
+}
+```
+
+### Create your own Fungible token
+
+```solidity
+// MyToken.sol
+// SPDX-License-Identifier: MIT
+
+import "@lukso/universalprofile-smart-contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol";
+
+pragma solidity ^0.8.0;
+
+contract MyToken is LSP7DigitalAsset {
+    // 4th argument (false) marks that this contract serves as a Fungible Token and not as a NFT.
+    constructor() LSP7DigitalAsset("MyToken","MTKN",msg.sender,false) {
+        // ..
+    }
+
+    function mint() public {
+        _mint(...);
+    }
+}
+```
+
+## Learn More
+
+- **[UniversalProfile & Identity Section](https://youtu.be/SbTo_e3l_Lk?t=1727)**
+- **[NFT 2.0 Section](https://youtu.be/hg1Ow6u9QVk)**
```

Comment:
What about putting a link with a specific time stamp in the video? So it starts right at the correct moment?